OctoKit/Issue.swift

```diff
@@ -58,25 +104,109 @@ public extension Octokit {
             }
         }
     }
+    
+    /**
+     Fetches an issue in a repository
+     - parameter owner: The user or organization that owns the repository.
+     - parameter repository: The name of the repository.
+     - parameter number: The number of the issue.
+     - parameter completion: Callback for the outcome of the fetch.
+     */
+    public func issue(owner: String, repository: String, number: Int, completion: (response: Response<Issue>) -> Void) {
```

Comment:
Given the url `https://api.github.com/repos/pietbrauer?page=2`.
How do you tell if the page parameter was a string or a number?